# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/tests/unit/test_trust_anchors/ee-notBefore-2023.pem.certspec
# Commit: dc509eab3ed7
# Full Hash: dc509eab3ed7a513258723eb46b25f33a610e720
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2025-07-11 16:36:21
# Regressor Bug: 1953221
# File Overlap Count: 3
# Description:
#   Bug 1953221 - don't accept SCTs with a timestamp after the distrustAfter date for the root r=jschanck
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D256720
# ==============================================================================

diff -r 5b86ae2c81f9 -r dc509eab3ed7 security/manager/ssl/tests/unit/test_trust_anchors/ee-notBefore-2023.pem.certspec
--- a/security/manager/ssl/tests/unit/test_trust_anchors/ee-notBefore-2023.pem.certspec	Thu Jul 10 18:37:49 2025 +0000
+++ b/security/manager/ssl/tests/unit/test_trust_anchors/ee-notBefore-2023.pem.certspec	Thu Jul 10 19:04:10 2025 +0000
@@ -2,3 +2,4 @@
 subject:ee
 validity:20230101-20510101
 extension:extKeyUsage:serverAuth
+extension:embeddedSCTList:default:20160101,secp256r1:20160101,alternate:20250101